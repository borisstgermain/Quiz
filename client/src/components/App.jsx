// @flow

import React from 'react';

export default function App() {
    return (
        <div className="app">
            Hello!! I'am APP!!11
        </div>
    )
}
