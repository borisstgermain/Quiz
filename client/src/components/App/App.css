.app {
  min-height: 100vh;

  background: linear-gradient(to bottom, #362057 60%, #3b4073);

  font-size: 100%;
}
