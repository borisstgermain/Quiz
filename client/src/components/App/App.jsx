// @flow

import React from 'react';
import style from './App.css';

import Quiz from '../../Quiz';

export default function App() {
  return (
    <div className={style.app}>
      <Quiz />
    </div>
  );
}
