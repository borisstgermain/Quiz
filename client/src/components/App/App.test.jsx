import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
    describe('Smoke tests', () => {
        test('Render', () => {
            shallow(<App />);
        });
    });

    describe('Component testing', () => {
        let wrapper = null;

        beforeEach(() => {
            wrapper = shallow(<App />);
        });

        test('Contains components', () => {
            expect(wrapper).toMatchSnapshot();
        });
    });
});
