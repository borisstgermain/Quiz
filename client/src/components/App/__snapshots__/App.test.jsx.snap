// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component testing Contains components 1`] = `
<div
  className="app"
>
  <Connect(Quiz) />
</div>
`;
